$NetBSD: patch-Makefile,v 1.3 2020/01/05 04:11:16 ryoon Exp $

Fix DashCast installation.
Remove non-standard cp(1) option.
Use VERSION_MAJOR also on Darwin (without git, VERSION is defined as UNKNOWN).

--- Makefile.orig	2019-06-27 10:20:45.000000000 +0000
+++ Makefile
@@ -159,17 +159,10 @@ endif
 
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/include"
 
-ifeq ($(CONFIG_DARWIN),yes)
 	cp $(SRC_PATH)/gui/icons/* "$(DESTDIR)$(prefix)/share/gpac/gui/icons/"
 	cp -R $(SRC_PATH)/gui/extensions/* "$(DESTDIR)$(prefix)/share/gpac/gui/extensions/"
 	cp $(SRC_PATH)/shaders/* "$(DESTDIR)$(prefix)/share/gpac/shaders/"
 	cp -R $(SRC_PATH)/include/* "$(DESTDIR)$(prefix)/include/"
-else
-	cp --no-preserve=mode,ownership,timestamp $(SRC_PATH)/gui/icons/* $(DESTDIR)$(prefix)/share/gpac/gui/icons/
-	cp -R --no-preserve=mode,ownership,timestamp $(SRC_PATH)/gui/extensions/* $(DESTDIR)$(prefix)/share/gpac/gui/extensions/
-	cp --no-preserve=mode,ownership,timestamp $(SRC_PATH)/shaders/* $(DESTDIR)$(prefix)/share/gpac/shaders/
-	cp -R --no-preserve=mode,ownership,timestamp $(SRC_PATH)/include/* $(DESTDIR)$(prefix)/include/
-endif
 
 lninstall:
 	$(INSTALL) -d "$(DESTDIR)$(prefix)"
